aws/templates/id/id_lb.ftl

```diff
@@ -126,34 +149,42 @@
         [#local fqdn = internalFqdn ]
         [#local scheme ="http" ]
     [/#if]
+
+    [#local url = scheme + "://" + fqdn  ]
+    [#local internalUrl = scheme + "://" + internalFqdn ]
+    
     [#return
         {
             "Resources" : {
                 "listener" : {
-                    "Id" : id,
+                    "Id" : listenerId,
                     "Type" : AWS_ALB_LISTENER_RESOURCE_TYPE
                 },
                 "sg" : {
-                    "Id" : formatDependentSecurityGroupId(id),
+                    "Id" : formatDependentSecurityGroupId(listenerId),
                     "Name" : core.FullName,
                     "Type" : AWS_VPC_SECURITY_GROUP_RESOURCE_TYPE
                 },
-                "targetgroups" : {
-                    "default" : {
-                        "Id" : formatALBTargetGroupId(occurrence, "default"),
-                        "Name" : formatName(core.FullName, "default"),
-                        "Type" : AWS_ALB_TARGET_GROUP_RESOURCE_TYPE
-                    }
+                "listenerRule" : {
+                    "Id" : formatResourceId(AWS_ALB_LISTENER_RULE_RESOURCE_TYPE, core.Id),
+                    "Type" : AWS_ALB_LISTENER_RULE_RESOURCE_TYPE
+                },
+                "targetgroup" : {
+                    "Id" : formatResourceId(AWS_ALB_TARGET_GROUP_RESOURCE_TYPE, occurrence.Core.Id),
+                    "Name" : formatName(core.FullName),
+                    "Type" : AWS_ALB_TARGET_GROUP_RESOURCE_TYPE
                 }
             },
             "Attributes" : {
                 "LB" : lbId,
                 "ENGINE" : engine,
                 "FQDN" : fqdn,
-                "URL" : scheme + "://" + fqdn,
+                "URL" : url + path,
                 "INTERNAL_FQDN" : internalFqdn,
-                "INTERNAL_URL" : scheme + "://" + internalFqdn,
-                "PORT" : sourcePort.Name
+                "INTERNAL_URL" : internalUrl + path,
+                "PORT" : sourcePort.Name,
```

Comment:
Think this is maybe where we need DESTINATION_PORT, and maybe change PORT to SOURCE_PORT for consistency